README.md

```diff
@@ -46,6 +46,8 @@ providers:
 
 ### Support Information
 
+Ultra provider supports full root NS record management.
+
```

Comment:
For consistency's sake other providers list this info in its own section after `#### Records`, https://github.com/octodns/octodns-ns1#root-ns-records

```
#### Root NS Records

Ns1Provider supports full root NS record management.
```

Tried to make this a suggestion, but it was out of context enough that it wouldn't let me.

Reply:
Gotcha, I moved it to follow the convention (I think)